src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -708,5 +933,22 @@ private XmlLiteralTextFormatDefinition()
             }
         }
         #endregion
+
+        #region Symbol - Static
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.StaticSymbol)]
+        [Name(ClassificationTypeNames.StaticSymbol)]
+        [Order(After = PredefinedClassificationTypeNames.Identifier)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class SymbolStaticFormatDefinition : ClassificationFormatDefinition
```

Comment:
❓ Can we also classify 'declaration' in a similar manner, so symbol references can be differentiated from symbol definitions if a user chooses?

Reply:
We _could_ but since our only font style option is Bold it is very limiting.

Reply:
I believe we may want to revisit this when compound classifications are supported by the editor and also the fonts and colors options.